modules/aggregations/src/test/java/org/elasticsearch/aggregations/bucket/histogram/InternalAutoDateHistogramTests.java

```diff
@@ -459,33 +457,6 @@ public void testCreateWithReplacementBuckets() {
         assertThat(copy.getInterval(), equalTo(orig.getInterval()));
     }
 
-    public void testSerializationPre830() throws IOException {
-        // we need to test without sub-aggregations, otherwise we need to also update the interval within the inner aggs
-        InternalAutoDateHistogram instance = createTestInstance(
-            randomAlphaOfLengthBetween(3, 7),
-            createTestMetadata(),
-            InternalAggregations.EMPTY
-        );
-        TransportVersion version = TransportVersionUtils.randomVersionBetween(
-            random(),
-            TransportVersions.MINIMUM_COMPATIBLE,
-            TransportVersionUtils.getPreviousVersion(TransportVersions.V_8_3_0)
-        );
-        InternalAutoDateHistogram deserialized = copyInstance(instance, version);
-        assertEquals(1, deserialized.getBucketInnerInterval());
-
-        InternalAutoDateHistogram modified = new InternalAutoDateHistogram(
-            deserialized.getName(),
-            deserialized.getBuckets(),
-            deserialized.getTargetBuckets(),
-            deserialized.getBucketInfo(),
-            deserialized.getFormatter(),
-            deserialized.getMetadata(),
-            instance.getBucketInnerInterval()
-        );
-        assertEqualInstances(instance, modified);
```

Comment:
This test can no longer pass, as the locale of the formatter changes from root to english, and that can't be easily fixed up afterwards

Reply:
I am not familiar with this test but I would wonder if we need a replacement of some sort for it. This is a question for the @elastic/es-analytical-engine I think . cc @nik9000